pkgs/development/libraries/dirac/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchurl, doxygen }:
+
+stdenv.mkDerivation rec {
+  version = "1.0.2";
+  name = "dirac-${version}";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/dirac/${name}.tar.gz";
+    sha256 = "1z803yzp17cj69wn11iyb13swqdd9xdzr58dsk6ghpr3ipqicsw1";
+  };
+
+  buildInputs = [ doxygen ];
+  enableParallelBuilding = true;
+
+  patches = [ ./dirac-1.0.2.patch ];
```

Comment:
@fpletz, yes, this patch is needed to fix compiler warnings that are treated as errors due to configuration settings. Configuration script does not allow to disable warnings-as-errors setting and I had to sed all mentions of -pedantic -Wall -Werror in the project or fix compiler warnings. I chose the second approach and prepared this patch. Is this not a common practice?